core/storage/statistics/gae_models.py

```diff
@@ -1071,6 +1162,13 @@ class PlaythroughModel(base_models.BaseModel):
     # ordered by the time of occurence of the action.
     actions = ndb.JsonProperty(repeated=True)
 
+    @staticmethod
+    def get_deletion_policy():
+        """Playthroughs are anonymized,
+        and cannot be tied back to an individual user.
```

Comment:
Nit: here and elsewhere, wrap to 80-chars (I think you have a line break too early).

Reply:
Done